summingbird-online/src/test/scala/com/twitter/summingbird/online/PlannerSpec.scala

```diff
@@ -94,14 +119,16 @@ class PlannerSpec extends WordSpec {
       .sumByKey(store2)
 
     val planned = Try(OnlinePlan(tail))
-    val path = TopologyPlannerLaws.dumpGraph(tail)
+
 
     planned match {
-      case Success(graph) => assert(true == true)
+      case Success(graph) => {
+        assert(true == true)
```

Comment:
you never need this. Just put `()` if you don't want to do anything (return `Unit`).